packages/sdk/contractkit/src/wrappers/Accounts.test.ts

```diff
@@ -67,18 +67,33 @@ testWithGanache('Accounts Wrapper', (web3) => {
   test('SBAT authorize attestation key', async () => {
     const account = accounts[0]
     const signer = accounts[1]
-    await accountsInstance.createAccount()
+    await accountsInstance.createAccount().sendAndWaitForReceipt({ from: account })
     const sig = await getParsedSignatureOfAddress(account, signer)
-    await accountsInstance.authorizeAttestationSigner(signer, sig)
+    await (await accountsInstance.authorizeAttestationSigner(signer, sig)).sendAndWaitForReceipt({
+      from: account,
+    })
+    const attestationSigner = await accountsInstance.getAttestationSigner(account)
+    expect(attestationSigner).toEqual(signer)
   })
 
   test('SBAT remove attestation key authorization', async () => {
     const account = accounts[0]
     const signer = accounts[1]
-    await accountsInstance.createAccount()
+    await accountsInstance.createAccount().sendAndWaitForReceipt({ from: account })
```

Comment:
Could you add `sendAndWaitForReceipt` to the tests below too? It should be used everywhere the tests try to modify chain state in preparation for the transaction they're testing (I.e. all the calls to `await accountsInstance.createAccount()` and any other calls on `accountsInstance` other than `getParsedSignatureOfAddress` which is a view call (doesn't modify state). These tests should be doing that already, but someone didn't know. Sorry about that :/

Reply:
Yep, sounds good. I'll add expect statements too